### PR TITLE
ch: add private route component

### PR DESCRIPTION
**Pull request by aapetsi**
## Description
Describe change or new feature here.
## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by aapetsi** [comment]
[Some route should not be accessed when you are not logged in](https://trello.com/c/rzHuQv2E/57-some-route-should-not-be-accessed-when-you-are-not-logged-in)

File: src/App.js [hunk 1]
```diff
@@ -18,30 +19,16 @@ function App() {
       <Route path='/register'>
         <WrappedRegistrationForm />
       </Route>
-      <Route path='/inventory'>
-        <Main />
-      </Route>
+      <PrivateRoute path='/inventory' component={Main} />
```
Comment (shaunorpen) [inline src/App.js:22]: Can we use the same format as the other routes for this?

`<PrivateRoute path='/inventory' component={Main} />`

becomes 

`<PrivateRoute path='/inventory'>
     <Main />
</PrivateRoute>`

**Review by shaunorpen — APPROVED** [review]
Happy to approve, one suggestion to consider.

File: src/App.js (hunk 1 above)
Comment (aapetsi) [reply src/App.js:22]: I tried that and it doesn’t work.

Comment (shaunorpen) [reply src/App.js:22]: Interesting. If that's the case I'm happy to close this issue.

**Pull request merged**